README.md

```diff
@@ -16,17 +15,29 @@ You need node.js for compilation and running of JavaScript specs
 `brew install node` if you don't has it,
 then `npm install -g grunt-cli` -- the grunt-cli may require sudo.
 
-Then, from the root `chiscore-two` directory, `npm install`
+Then, from the root `chiscore` directory, `npm install`
+
+## Setup and Server
+Navigate to the  root of the `chiscore` directory:
+    `cd chiscore`
```

Comment:
spaces and backticks seem a bit redundant to me.  I'd do the 4 spaces for single-lined commands.